main.tf

```diff
@@ -400,7 +401,8 @@ module "extensions" {
   nodepool_upgrade_method = var.nodepool_upgrade_method
   node_pools_to_drain     = var.node_pools_to_drain
 
-  debug_mode = var.debug_mode
+  debug_mode        = var.debug_mode
+  update_kubeconfig = var.update_kubeconfig
```

Comment:
I was actually thinking of deleting the local kubeconfig. What's your view on this? keep or do not keep?

Reply:
I don't use it - in favor of removal.

Reply:
Ok, go ahead then.

Reply:
We decided to leave for a separate PR.